config/application.rb

```diff
@@ -32,5 +32,8 @@ class Application < Rails::Application
     config.generators do |g|
       g.orm :active_record, primary_key_type: :uuid
     end
+
+    config.active_job.queue_adapter = :que
+    config.action_mailer.deliver_later_queue_name = ""
```

Comment:
I read that this should be `"default"` is that implied somehow?

Reply:
Yeah, you're right. `:default` works as well, I've changed it so it's more obvious.